app/src/main/resources/config/application.yml

```diff
@@ -59,14 +59,20 @@ valtimo:
             secret: e09b8bc5-5831-4618-ab28-41411304309d
             rsin: "051845623"
             documentTypeUrl: http://localhost:8001/catalogi/api/v1/informatieobjecttypen/efc332f2-be3b-4bad-9e3c-49a6219c92ad
-        documentenapi:
-            url: http://localhost:8001
-            clientId: valtimo_client
-            secret: e09b8bc5-5831-4618-ab28-41411304309d
         zakenapi:
             url: http://localhost:8001
             clientId: valtimo_client
             secret: e09b8bc5-5831-4618-ab28-41411304309d
+        documentenapis:
+            documentapis:
```

Comment:
The path here is a bit strange. I would either include the list of configuration under `documentenapis` or change it to `documentenapis.configurations` instead of the current `documentenapis.documentapis`

Reply:
@ivo-ritense agreed; directly under documentapis isnt possible so changed it to configurations